plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -51,13 +51,14 @@ type GRPCEnforcementPolicy struct {
 // CiscoTelemetryMDT plugin for IOS XR, IOS XE and NXOS platforms
 type CiscoTelemetryMDT struct {
 	// Common configuration
-	Transport         string
-	ServiceAddress    string                `toml:"service_address"`
-	MaxMsgSize        int                   `toml:"max_msg_size"`
-	Aliases           map[string]string     `toml:"aliases"`
-	Dmes              map[string]string     `toml:"dmes"`
-	EmbeddedTags      []string              `toml:"embedded_tags"`
-	EnforcementPolicy GRPCEnforcementPolicy `toml:"grpc_enforcement_policy"`
+	Transport          string
```

Comment:
When we are at it, can you please also add the (unrelated) missing TOML annotation here?

Reply:
Done: [8fe6d33](https://github.com/influxdata/telegraf/pull/12345/commits/8fe6d33ef5ad0fe4f4f5911b4cdbf447bd80cc46)